<launch>
  <arg name="serial_port" default="/dev/ttyUSB0" />
  <arg name="baud_rate" default="115200" />

  <node name="car_motherboard" pkg="ucl_ddcar" type="car_motherboard.py">
    <param name="serial_port" value="$(arg serial_port)" />
    <param name="baud_rate" value="$(arg baud_rate)" />
  </node>
</launch>

